java-checks-test-sources/default/src/main/java/checks/S2201_IgnoredReturnValueCheckSample.java

```diff
@@ -176,4 +176,8 @@ public Collection<String> get() {
   List<String> makeList() {
     return new ArrayList<>();
   }
+
+  void validateValue(Object o){
+    Optional.ofNullable(o).orElseThrow(); //FP here orElseThrow() is used for the side effect
```

Comment:
Let's make the comment more helpful for the future.
```suggestion
    Optional.ofNullable(o).orElseThrow(); // Compliant: orElseThrow() is used for the side effect
```

Reply:
:+1: